server/routes/referrals/completionDeadlinePresenter.ts

```diff
@@ -15,17 +15,20 @@ export default class CompletionDeadlinePresenter {
 
   readonly errorSummary: CompletionDeadlineErrorSummaryPresenter | null
 
-  readonly title = `What date does the ${this.referral.serviceCategory.name} service need to be completed by?`
+  readonly title = `What date does the ${this.serviceCategory.name} service need to be completed by?`
 
   readonly hint = 'For example, 27 10 2021'
 
   constructor(
     private readonly referral: DraftReferral,
+    private readonly serviceCategory: ServiceCategory,
     errors: CompletionDeadlineErrors | null = null,
     userInputData: Record<string, unknown> | null = null
   ) {
     if (!userInputData) {
-      const calendarDay = referral.completionDeadline ? CalendarDay.parseIso8601(referral.completionDeadline) : null
+      const calendarDay = this.referral.completionDeadline
```

Comment:
This looks unrelated to the rest of the changes, was it intentional?

Reply:
Yeah, although maybe it should have been in its own commit. I may be misremembering how Typscript instance variables work, but how did this work before? My editor highlighted `referral.completionDeadline` as an error, how come it wasn't `this.referral.completionDeadline` and did that not throw errors for you?

Reply:
I think it worked because `referral` is also a parameter of the constructor so it's available inside the constructor as a local variable, as any other method parameter would be? That's my guess.

Reply:
Right, well now my editor isn't complaining about when I put it back, so have undone this (although prettier still split it onto multiple lines)